widgetpanel.go

```diff
@@ -202,7 +206,7 @@ func (w *widgetPanel) setBrightness(diff int) {
 	if err != nil {
 		log.Println("Error running xbacklight", err)
 	} else {
-		w.brightness.SetValue(brightness())
+		w.brightness.SetValue(floatVal)
```

Comment:
I tested this and every time the plus or minus buttons is pressed the change is made but the old value is displayed.
I think the reason is that `value` should be used here, not `floatVal`.
Perhaps value should be `newValue` or something to make this clearer?